### ISSUE TITLE
Scrollbars are displayed over the text

### ISSUE THREAD

**Issue by marbaa** [quodlibet/quodlibet#3521]
Hi there,

Just look at pictures:
![Screenshot_20201211_111139](https://user-images.githubusercontent.com/40859847/101891268-1bad1200-3ba2-11eb-9ce6-bd67179fb810.png)
![Screenshot_20201211_111355](https://user-images.githubusercontent.com/40859847/101891272-1c45a880-3ba2-11eb-80e1-f5c2606f33bd.png)

### Which version of Quod Libet?
4.3.0

### Which operating system 
Neon 20.04 KDE 5.20

**Comment by marians** [comment]
I had a similar problem on MacOS, but can't reproduce it currently. Watching out for it to re-appear.

**Comment by CreamyCookie** [comment]
Isn't this a theming issue / non issue? At least, I thought that this is the very idea of overlay scrollbars.. (that they overlay when you need them, but then disappear to make space for other things)

**Comment by bassstorm** [comment]
Yeah, but perhaps a little problem here is that scrollbar appears overlay _all the time_ you work with playlist. I.e. you don't scroll, you don't move pointer close to the right border... but it's over the text which is subject to read (hard to explain, but that's maybe because I keep mouse ready for selection while looking for the right line). Luckily it's not that big in my KDE theme like on screenshots, but still noticeable.

**Comment by CreamyCookie** [comment]
> overlay all the time you work with playlist. I.e. you don't scroll, you don't move pointer close to the right border

Ah, got it! Yeah, that's an issue that should be addressed if it's QL's fault.

**Comment by marbaa** [comment]
Those scrollbars are there always, not only when I'm, working with playlist, even when app has no focus. I don't know what is the trend to mess with scrollbars last years, on every side I watch, doesn't matter if Linux/Windows/Mac. Making them thinner, removing up/down arrows... Always braking something which was working fine for decades.

**Comment by CreamyCookie** [comment]
@marbaa I agree. Apps shouldn't even have control over how the scrollbar looks. The scrollbars changes should be completely optional (like giving an app access to certain permissions it doesn't need). 

The whole thing reminds me of the problems of [Client-Side Decorations](https://github.com/quodlibet/quodlibet/issues/2297#issuecomment-504660657), where the app handles the window frame rather than the window manager (as it has been for a long time with Server-Side Decorations).

**Comment by nagisa** [comment]
It seems like its the standard GTK behaviour to reveal the overlay scrollbar as soon as ScrolledWindow has any mouse movement over it. As far as system-wide settings go, you can disable these overlay scrollbars by setting `overlay-scrolling` property in `org.gnome.desktop.interface` gsettings as such:

```
gsettings set org.gnome.desktop.interface overlay-scrolling false
```

As far as application side of things go, I think its mostly caused by the fact that quodlibet is very likely to put a lot of interesting information very close to the right-hand side (left-hand side /w RTL) of the `ScrolledWindow`s. Most other applications I've seen around either don't really have anything this close to the side which has a scrollbar or they use a fair amount of padding to give some space.

**Comment by declension** [comment]
Thanks @nagisa. I'd be happy increasing the horizontal padding a bit as one mitigation
